### DISCUSSION TITLE
feature: display quantity of items within the hotbar

### DISCUSSION THREAD

**Discussion by Gorgaki** [AscensionGameDev/Intersect-Engine#2300] [Ideas]
I'm thinking of a very simple feature that could improve the player experience.
Display the quantities of items owned in the hotbar.
![hotbar1](https://github.com/AscensionGameDev/Intersect-Engine/assets/83343437/da76aafe-26ff-4576-bb6f-e3cbac7cfd81)

**Comment by EspectroW94** [comment]
I was just thinking about proposing this idea, I think it would be very useful.

**Comment by Arufonsu** [comment]
Added to [_(v0.8.0-beta.120+) Roadmap_](https://github.com/AscensionGameDev/Intersect-Engine/discussions/2303)

**Comment by Arufonsu** [comment]
solved by #2342
